src/cryptonote_protocol/cryptonote_protocol_handler.inl

```diff
@@ -336,6 +337,32 @@ namespace cryptonote
     LOG_INFO_CC(context, "New connection posing as pruning seed " << epee::string_tools::to_string_hex(context.m_pruning_seed) << ", seed address " << &context.m_pruning_seed);
 #endif
 
+    // Check for any blink txes being advertised that we don't know about
```

Comment:
I'm not sure that adding blink to the core_sync_data is appropriate. This is for when you're behind and syncing the chain. In which case you don't care about blinks, only at the tip of the chain do you care about blinks and transactions floating around in the transaction pool

blink_txs also isn't getting filled out yet?

Reply:
Right, this is only half complete because I'm not sure about it either and need to think some more about how to go about this.

My thinking on the core sync approach was to include a fairly small set of data here: initially (just for simplicity) a set of txids of active and recent blink transactions, but as blink txes pick up, changing this to something along the lines of:

{ 1234 => HASH, 1235 => HASH, ..., mempool => HASH } where each HASH is a hash of all the blink TXs at that height.  The amount of data would be fairly small, but the local node can then easily tell that its local blinks at some recent height don't agree and exchange blinks at the given height with the peer.  (Heights would go back to the immutable block).

Reply:
(Comments on the above approach or possible alternatives would be appreciated, of course).

Reply:
Implemented in PR #927

Reply:
Implemented in #927